src/lib/y2users/users_simple/reader.rb

```diff
@@ -34,18 +34,25 @@ def read_to(config)
         users = Yast::UsersSimple.GetUsers
         # TODO: only created users, not imported ones for now
         users.each do |user|
-          user = User.new(config, user["uid"], gecos: [user["cn"]])
+          user = User.new(user["uid"])
```

Comment:
We're overwriting _user_ here, which is going to raise am `undefined method "[]" for #<Y2Users::User....`